client/src/components/generic/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+
+type DropdownProps = React.SelectHTMLAttributes<HTMLSelectElement>
+
+const Dropdown = ({ ...props }: DropdownProps) => {
+  return (
+    <select
+      className="flex-shrink:0 border-radius: 0.25rem h-9 w-full rounded"
+      style={{
+        borderColor: "var(--Greys-100, #242424)",
+        borderWidth: "1px",
+        borderStyle: "solid",
+        paddingLeft: "18px",
+        WebkitAppearance: "none",
+        MozAppearance: "none",
```

Comment:
also nit: tailwind (compile time) is preferred over styled prop (runtime) because of better performance